docs/source/en/api/models/transformer2d.mdx

```diff
@@ -0,0 +1,29 @@
+# Transformer2D
+
+A Transformer model for image-like data. [`Transformer2DModel`] accepts discrete (classes of vector embeddings) or continuous (actual embeddings) inputs.
+
+When the input is **continous**:
+
+1. Project the input and reshape it to `(b, t, d)`.
+2. Apply standard Transformer action.
```

Comment:
```suggestion
2. Apply the Transformer blocks in the standard manner.
```

Reply:
Maybe also include some examples where continuous embeddings are used as opposed to discrete would be nice.